src/Nncase.Tests/Evaluator/UnitTestEvaluatorMath.cs

```diff
@@ -0,0 +1,1033 @@
+// Copyright (c) Canaan Inc. All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Autofac;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NetFabric.Hyperlinq;
+using Nncase.Evaluator;
+using Nncase.IR;
+using Nncase.IR.F;
+using Nncase.IR.Math;
+using Nncase.IR.Tensors;
+using Nncase.TestFixture;
+using Nncase.Utilities;
+using OrtKISharp;
+using Xunit;
+using static Nncase.IR.F.Math;
+using static Nncase.IR.F.NN;
+using static Nncase.IR.F.Tensors;
+using static Nncase.Utilities.DumpUtility;
+using RangeOf = Nncase.IR.Math.RangeOf;
+using Tuple = Nncase.IR.Tuple;
+
+namespace Nncase.Tests.EvaluatorTest;
+
+public class UnitTestEvaluatorMath : TestFixture.UnitTestFixtrue
+{
+    [Fact]
+    public void TestBinaryScalarScalar()
+    {
+        var ops = new BinaryOp[]
```

Comment:
you can use EnumValues to get all the values of the enumeration

Reply:
Not all of binary ops are implemented at present.